packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -111,6 +196,12 @@ class MemoryController extends DisposableController
     _paused = false;
   }
 
+  bool _toggleAndroidChart = false;
```

Comment:
rename to _androidChartVisible

Reply:
done as well as isAndroidChartVisible and toggleAndroidChartVisibility.